email-generator/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:alpine3.13
```

Comment:
Fernando does not like Alpine. Please change base to UBI and the newer lightweight UBI msgs - ubi-lite? This is explicitly meant to replace Alpine based images. Either this or plain old ubi8